### ISSUE TITLE
[css-tables] Zero-width spaces in border-spacing dfn

### ISSUE THREAD

**Issue by tidoust** [w3c/csswg-drafts#2918]
The value definition of the `border-spacing` property has zero-width spaces around `length`, which makes the value hard to parse following the [Value Definition Syntax](https://drafts.csswg.org/css-values-4/#value-defs). By definition, that is difficult to see, but these spaces exist in the [source file](https://github.com/w3c/csswg-drafts/blob/master/css-tables-3/Overview.bs#L803)

**Comment by dontcallmedom** [comment]
I believe this was fixed in https://github.com/w3c/csswg-drafts/commit/39fd0aaafdee37b4591cfa11c9c5d253943e9f99#diff-6e3717cf0173351cda3264b270e9586a
